code/ensemble/README.md

```diff
@@ -1,7 +1,35 @@
-### Create an equally weighted ensemble (currentl only Germany, national level)
+## European hub ensembles
 
-Steps to create the ensemble forecast:
+The European hub produces weekly ensemble forecasts from valid submitted models. Ensembles are saved in [data-processed](./data-processed) with the team name "EuroCOVIDhub".
 
-- Create a new file `included_models/included_models-<forecast_date>.csv` containing info on which models to include. This serves to keep things reproducible.
-- Run the R script `simple_ensemble.R` (with working directory set to the containing folder).
-- Forecasts are written to `data-processed/KITCOVIDhub-ensemble` and can sent to the repo via PR.
+### Inclusion criteria
+
+Forecasts must have all of the following to be included in the ensemble:
+
+- Includes all quantiles for each horizon
+- Includes forecasts over a four week horizon
+- Not manually specified for exclusion (e.g. because of late submission)
+
+We detail the inclusion and exclusion of models in a [weekly csv](./code/ensemble/weekly-criteria).
+
+### Ensemble methods
+
+**Current methods**
+
+Ensembling methods combine forecast values by target, location, horizon, and quantile. At the moment we use the following:
+
+- Mean average ([`EuroCOVIDhub-ensemble`](./data-processed/EuroCOVIDhub-ensemble))
+- Median average ([`EuroCOVIDhub-median`](./data-processed/EuroCOVIDhub-median))
+
+The ensemble we use in evaluation is the "EuroCOVIDhub-ensemble".
```

Comment:
I'd add something like: "We are continually reviewing the performance of the default ensemble compared to other ensembling methods and will make a change if we find a different method to consistently outperform the current default" or something to that effect.